frontend/src/components/Profile/Profile.js

```diff
@@ -0,0 +1,88 @@
+import Axios from 'axios';
+import { Link } from "react-router-dom";
+import React, { useState, useEffect } from 'react'
+import '../../config';
+import './Profile.scss'
+
+export default function Profile(props) {
+    const [username, setUsername] = useState("");
+    const [biography, setBiography] = useState("");
+    const [displayName, setDisplayName] = useState("");
+    const [numFollowers, setNumFollowers] = useState(undefined);
+    const [numFollowings, setNumFollowings] = useState(undefined);
+    const [numPictures, setNumPictures] = useState(undefined);
+
+    useEffect(() => {
+        const loadProfile = () => {
+            Axios.get(global.config.BACKEND_URL + `/account/profile/${props.username}`).then(
+                (response) => {
+                    setUsername(response.data.username);
+                    setBiography(response.data.biography);
+                    setDisplayName(response.data.displayName);
+                    setNumFollowers(response.data.numFollowers);
+                    setNumFollowings(response.data.numFollowings);
+                    setNumPictures(response.data.numPictures);
+                }).catch((error) => {
+                    console.log(error.response);
+                })
+        }
+        loadProfile();
+    }, [props])
+
+    return (
+        <div className="profile-container">
+            <table className="profile">
```

Comment:
From what I know, tables aren't responsive and tend to break when changing device (from laptop to tablet to mobile, for example). My advice to you is to go for divs and use `display: flex;`. If you need help, let me know!